newsfragments/3529.feature.rst

```diff
@@ -0,0 +1 @@
+Prevents nodes from starting up or participating in DKGs if there is a local vs. onchain ferveo key mismatch.  This will assist in alerting node operators who need to relocate or recover thier hosts about the correct procedure.
```

Comment:
```suggestion
Prevents nodes from starting up or participating in DKGs if there is a local vs. onchain ferveo key mismatch.  This will assist in alerting node operators who need to relocate or recover their hosts about the correct procedure.
```